.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
```

Comment:
indent size should be 4, as per [hapi's style guide](https://github.com/hapijs/contrib/blob/master/Style.md#whitespace)